src/Commands/ToggleAutoShifting.cpp

```diff
@@ -0,0 +1,21 @@
+#include "ToggleAutoShifting.h"
+#include "TankDrive.h"
+#include "../Subsystems/Chassis.h"
+
+ToggleAutoShifting::ToggleAutoShifting()
+: mAutoShift(nullptr) {
```

Comment:
`mAutoShift` does not need to be a member variable. The value is never set except in initialize.

You can remove `mAutoShift` from the class.